odoo/addons/test_read_group/tests/test_formatted_read_group.py

```diff
@@ -2033,16 +2033,203 @@ def setUpClass(cls):
         cls.usd.active = True
         cls.eur = cls.env.ref('base.EUR')
         cls.eur.active = True
+        cls.stn = cls.env.ref("base.STN")
+        cls.eur.active = False
```

Comment:
Why activating `cls.eur` then deactivating it?
Doesn't this impact existing tests?